1904labs/de-hours-with-experts
# Paullentz challenge three

Wrote a function to decode a string, using that function wrote a function to decode a line of text.
Then used those functions in the main method to read all the ingredients from secret_recipe.txt and write decoded ingredients to a new file called decoded_recipe.txt saved in the same /de-hours-with-experts/python/ directory as the code. 

Testing steps:
1. Delete the current decoded_recipe.txt file in the /de-hours-with-experts/python/ directory.
<img width="740" alt="image" src="https://github.com/1904labs/de-hours-with-experts/assets/118093209/786bb8b6-ce0a-4757-945c-20ca46324be4">


2. Run the code in the file secret_recipe_decoder.py located in the /de-hours-with-experts/python directory by right-clicking the file name or right-clicking within the file and choosing:
"Run Python - Run Python File in Terminal"
<img width="777" alt="image" src="https://github.com/1904labs/de-hours-with-experts/assets/118093209/abd0a9f0-8add-45bc-b10d-93c9a028fbd1">

3. Look to see a newly created decoded_recipe.txt  in the same directory and see the decoded recipe.
<img width="737" alt="image" src="https://github.com/1904labs/de-hours-with-experts/assets/118093209/cc1e8646-ea1a-4ee4-9015-f62ecb8eaaef">
